app/components/TrackComponent/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ *
+ * TrackComponent
+ *
+ */
+
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { isEmpty } from 'lodash';
+import { colors } from '@app/themes/index';
+import If from '@components/If';
+import { Card, Image, Typography, Button } from 'antd';
+
+const { Title, Paragraph } = Typography;
+
+const TrackCardContainer = styled(Card)`
+  && {
+    border-radius: 0.5rem;
+    width: ${props => (props.width ? props.width : '25rem')};
+    border: none;
+    background-color: ${colors.secondary};
+    text-align: center;
+`;
+
+const StyledDescription = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const PlayTrackBtn = styled(Button)`
+  && {
+    background-color: ${colors.btnBackground};
```

Comment:
So the naming should not be like this. You need to name the color. Because you might need the color at multiple places.

Lets take this case. What if you needed the same color shade for button as well as for a card. Would you use colors.btnBackground for card as well ? Even if you create a new variable for card like cardBackground that points to the same color shade you are just duplicating things. So define the color and use it at multiple places

Reply:
updated the changes